core/storage/statistics/gae_models.py

```diff
@@ -62,6 +62,8 @@
     'MultipleIncorrectSubmissions': 'state_name',
     'CyclicStateTransitions': 'state_names'
 }
+# The choices of entity type valid for LearnerAnswerDetailsModel.
+ENTITY_TYPE_CHOICES = [feconf.ENTITY_TYPE_EXPLORATION]
```

Comment:
ALLOWED_ENTITY_TYPES

Reply:
Also, what about question? Your PR needs to correctly handle both explorations and questions.

Reply:
Done